javascript/react/props-iii/shallow-compare-in-react.md

```diff
@@ -61,7 +61,7 @@ The **NotPure** component would render every time **this.props.className** updat
 
 Note that `PureComponent` only shallowly compares the objects, meaning it the **prop**s or **state** contains complex data structures, it may produce false-negatives for deeper differences. Only use `PureComponent` if the component has simple **prop**s and **state**.
```

Comment:
Just noticed a mistake. It should say "meaning _if_ the **prop**s" not "meaning _it_ the **prop**s"

Reply:
nice spot 👍